pyproject.toml

```
@@ -27,10 +27,8 @@ classifiers = [
 ]
 requires-python = '>=3.10'
 dependencies = [
-   'imagecodecs',
    'numpy',
-   'tifffile>=2023.9.26',
-   'zarr>=2,<3',
+   'tifffile[codecs, zarr]',
```

I do think we should set a minimum tifffile version?

```suggestion
   'tifffile[codecs, zarr]>=2023.9.26',
```

---

So those groups don't go back that far.  Wasn't sure how it would work.

---

Ok, so yeah that won't work because that version didn't have those optionals. But I don't think we need to support that far back, since we dont support py39 and *do* support py312?

---

I opted for 2024.7.21 which has the same python version compatibility as napari.